### DISCUSSION TITLE
How do you pronounce Hono?

### DISCUSSION THREAD

**Discussion by kerolloz** [honojs/hono#2335] [General]
Some people say it 'Hono' others say it 'Ono', which is correct?

**Comment by yusukebe** [comment]
Hi @kerolloz 

Hono is "**Hono**"! 

I think it's "***hono'o***" to be more specific.

**Comment by ryuujo1573** [comment]
It's `/hɒ nɒ o/` 😄

**Comment by MohsinDev369** [comment]
Since it's taken from the Japanese word Hono meaning Fire, Pronounce it using diaphragm.

**Comment by RickCogley** [comment]
I'm a long term Japan resident and Japanese speaker, here because I keep hearing people pronounce it "hah-noe" which gives me a rash, lol. Japanese language is simple in terms of its vowels - they are always basically the same and there are no troublesome dipthongs like English has. 

Hono is pronounced close to English "oh no" with an h in front, and conversely is _not_ like "hah noe". There's a light accent on the first syllable "ho", and the "no" is drawn out a little, as @yusukebe illustrates via their more detailed spelling hono'o. If you have a mac, you can change your system language to Japanese and let the kyoko voice say it: 

```
> say "炎"
```

https://github.com/user-attachments/assets/5e4719e1-f2a8-49f9-ab73-1283f1cbfa85
